.github/workflows/publish_docker_image.yml

```diff
@@ -36,3 +37,19 @@ jobs:
             --quiet \
             --config=${{inputs.config_file}} \
             --substitutions=TAG_NAME=${{github.ref_name}} .
+
```

Comment:
The only thing I'm wondering is whether we need some delay here so the images are actually available and ready to pull from GCP? I know that we also sleep for a few minutes after the PyPI release before we start the docker builds because we ran into issues before.

Reply:
Probably can't hurt.

Reply:
Agreed. I added the sleep job before sending the dispatch event. Thanks!